README.md

```diff
@@ -43,7 +92,7 @@ A non-zero exit code and error message will be returned for any failures:
     Tasks: TOP => syntax => syntax:manifests
     (See full trace by running task with --trace)
 
-Use in conjunction with lint and spec tests for Continuous Integration:
+For Continuous Integration, use Puppet::Syntax in conjunction with `puppet-lint` and spec tests:
```

Comment:
What is this code sample? It doesn't look like command line stuff; is it something that goes in the Rakefile?